### PR TITLE
Hops pypi package ready

### PR DESCRIPTION
**Pull request by eirannejad**
- code cleanup
- package prep
- now works with `rhinoinside`

### CODE REVIEW

**Comment by eirannejad** [comment]
Resolved merge conflict on `.gitignore`

File: src/ghhops-server-py/.vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "files.exclude": {
+        "**/__pycache__": true
+    },
+    "python.pythonPath": "/Users/ein/.local/share/virtualenvs/ghhops-server-py-aHT-T4wI/bin/python",
```
Comment (sbaer) [inline src/ghhops-server-py/.vscode/settings.json:5]: I'm not very familiar with this settings file. It has a path for your computer in it. Is this important?

File: src/hops/HopsComponent.cs [hunk 2]
```diff
@@ -245,7 +245,7 @@ protected override void Render(GH_Canvas canvas, System.Drawing.Graphics graphic
 
             void RenderHop(System.Drawing.Graphics graphics, int alpha, System.Drawing.PointF anchor)
             {
-                var boxHops = new System.Drawing.RectangleF(anchor.X - 14, anchor.Y - 8, 16, 16);
+                var boxHops = new System.Drawing.RectangleF(anchor.X - 16, anchor.Y - 8, 16, 16);
```
Comment (sbaer) [inline src/hops/HopsComponent.cs:248]: :) I kept playing with this too

**Review by sbaer — APPROVED** [review]

File: src/ghhops-server-py/.vscode/settings.json (hunk 1 above)
Comment (eirannejad) [reply src/ghhops-server-py/.vscode/settings.json:5]: Removed!

**Pull request merged**